src/index.html

```diff
@@ -14,6 +14,7 @@
     <script src="ext-libs/dialog-polyfill.js"></script>
     <script src="js/viz.js" type="text/javascript"></script>
     <script src="js/lightbeam.js" type="text/javascript"></script>
+    <script src="js/lightbeamVR.js" type="text/javascript"></script>
```

Comment:
nit: can remove the ` type="text/javascript"` from the past three (for consistency)